src/main/java/seedu/address/logic/commands/ListPersonCommand.java

```diff
@@ -0,0 +1,26 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.model.Model.PREDICATE_SHOW_ALL_PERSONS;
+import static seedu.address.model.Model.PREDICATE_SHOW_NO_CONCERTS;
+
+import seedu.address.model.Model;
+
+/**
+ * Lists all persons in the address book to the user.
+ */
+public class ListPersonCommand extends Command {
+
+    public static final String COMMAND_WORD = "listp";
+
+    public static final String MESSAGE_SUCCESS = "Listed all persons";
+
+
+    @Override
+    public CommandResult execute(Model model) {
+        requireNonNull(model);
+        model.updateFilteredConcertList(PREDICATE_SHOW_NO_CONCERTS);
```

Comment:
Same here. List person should not update concert list

Reply:
understood, since we might need to filter or find a person or contact while listing the other one